gcloud/logging/client.py

```diff
@@ -79,7 +80,8 @@ def _entry_from_resource(self, resource, loggers):
         raise ValueError('Cannot parse job resource')
 
     def list_entries(self, projects=None, filter_=None, order_by=None,
-                     page_size=None, page_token=None):
+                     page_size=None,
+                     page_token=None):
```

Comment:
Dunno.  Might've come about while resolving conflicts with the `list_sinks` stuff?